lib/logstash/outputs/solr_http.rb

```diff
@@ -71,7 +71,7 @@ def flush(events, close=false)
         documents.push(document)
     end
 
-    @solr.add(documents)
+    @solr.add(documents, :add_attributes => {:commitWithin=>10000})
```

Comment:
It is better to configure the commitWithin on the collection instead of per document. This allows Solr to better control when commits are happening. If you need this here it should at least be configurable.